File: coding-test/백준/주사위세개.js
// https://www.acmicpc.net/problem/2480

/**
 * @Date 2024.02.14
 */

const fs = require('fs');
const [a, b, c] = fs.readFileSync('/dev/stdin').toString().split(' ').map(Number);
// const [a, b, c] = [3, 3, 6];
// const [a, b, c] = [2, 2, 2];
// const [a, b, c] = [6, 2, 5];

if (a === b && b === c) {
  console.log(10000 + a * 1000);
  return;
}
if (a === b) {
  console.log(1000 + a * 100);
  return;
}
if (b === c) {
  console.log(1000 + b * 100);
  return;
}
if (a === c) {
  console.log(1000 + a * 100);
  return;
}
console.log(Math.max(a, b, c) * 100);

File: coding-test/백준/기본/주사위세개.py
# https://www.acmicpc.net/problem/2480

# 2024.02.14

import sys

a, b, c = list(map(int, sys.stdin.readline().split()))
# a, b, c = [3, 3, 6]
# a, b, c = [2, 2, 2];
# a, b, c = [6, 2, 5];

if(a==b and b==c):
    print(10000 + a * 1000)
elif(a==b):
    print(1000 + a * 100)
elif(b==c):
    print(1000 + b * 100)
elif(a==c):
    print(1000 + a * 100)
else:
    print(max(a,b,c) * 100)
